tests/Integration/Behaviour/behat.yml

```diff
@@ -348,6 +348,7 @@ default:
                 - Tests\Integration\Behaviour\Features\Transform\LocalizedArrayTransformContext
                 - Tests\Integration\Behaviour\Features\Transform\StringToArrayTransformContext
                 - Tests\Integration\Behaviour\Features\Transform\StringToBoolTransformContext
+                - Tests\Integration\Behaviour\Features\Context\Domain\TaxFeatureContext
```

Comment:
Please try to keep the alphabetical order in the context list, it reduces risks of collision from multiple PRs and it allows finding contexts more easily by search when needed This one should be a few lines above after `Tests\Integration\Behaviour\Features\Context\Domain\SupplierFeatureContext`